packages/client/src/containers/Login.container/SignupForm.js

```diff
@@ -9,14 +9,25 @@ const SignupForm = () => {
     password: '',
   });
   const [formErrors, setFormErrors] = useState({});
+  const [isSubmit, setIsSubmit] = useState(false);
+  const message = '';
+
   const handleChange = (e) => {
     setFormValues({ ...formValues, [e.target.name]: e.target.value });
   };
 
   const handleSubmit = (event) => {
+    setIsSubmit(true);
     event.preventDefault();
     setFormErrors(FormValidation(formValues));
   };
+  /* eslint-disable no-console */
+
+  useEffect(() => {
+    if (Object.keys(formErrors).length === 0 && isSubmit) {
+      console.log(formValues);
```

Comment:
do you really need to keep tracking of formValues on the console?

Reply:
One of the tasks is to log form content into the console. or should I do it differently?

Reply:
Ok, sorry I cannot check Jira. 
I just wondered so we don't have an overload in the console.